rlclientlib/vw_model/safe_vw.cc

```diff
@@ -326,8 +334,8 @@ mm::model_type_t safe_vw::get_model_type(const VW::config::options_i* args)
 }
 
 bool safe_vw::is_compatible(const std::string& args) const {
-  const auto local_model_type = get_model_type(_vw->options);
-  const auto inbound_model_type = get_model_type(args);
+  const auto local_model_type = get_model_type(args);
```

Comment:
why did we swap the parameters we passed in in lines 337, 338?

Reply:
The initial values were incorrect. It didn't affect functionality because there was only equality check.
local model is created from initial command line arguments. 
inbound model is created from vw model.